docs/installation.rst

```diff
@@ -130,7 +127,8 @@ Manually Prepared Environment
 
 Make sure all of fmriprep's `External Dependencies`_ are installed.
 These tools must be installed and their binaries available in the
-system's ``$PATH``.
+system's ``$PATH``. In particular, FreeSurfer requires a license
```

Comment:
Please start new sentences on new lines. (Particularly in larger paragraphs, this makes it easier to see changes in the diffs restricted to the relevant sentence.)